experimenter/experimenter/nimbus-ui/src/components/PageEditBranches/FormBranches/FormFeatureValue/fml.test.ts

```diff
@@ -0,0 +1,77 @@
+import { json } from "@codemirror/lang-json";
+import { EditorState, EditorStateConfig } from "@codemirror/state";
+import { basicSetup } from "codemirror";
+import { fmlLinter } from "src/components/PageEditBranches/FormBranches/FormFeatureValue/fml";
+import { simpleObjectSchema } from "src/components/PageEditBranches/FormBranches/FormFeatureValue/schema";
+import { getConfig_nimbusConfig_applicationConfigs_channels } from "src/types/getConfig";
+import { getExperiment_experimentBySlug_featureConfigs } from "src/types/getExperiment";
+import {
+  NimbusExperimentApplicationEnum,
+  NimbusExperimentChannelEnum,
+} from "src/types/globalTypes";
+import { z } from "zod";
+
+const SIMPLE_SCHEMA: z.infer<typeof simpleObjectSchema> = {
+  type: "object",
+  properties: {
+    foo: { type: "string" },
+    bar: { type: "integer" },
+    baz: { type: "boolean" },
+  },
+};
+
+const COMPLEX_SCHEMA = {
+  type: "object",
+  properties: {
+    foo: {
+      type: "array",
+      items: {
+        type: "object",
+        properties: {
+          bar: {
+            type: "string",
+          },
+        },
+      },
+    },
+  },
+};
```

Comment:
Not completely sure what this is for. Is this dead code?

Reply:
dead as a doornail. removing it 🔪